doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -109,14 +84,10 @@ for more information.
      This indicates that you're likely picking up the wrong SciPy install,
      check with ``python -c "import scipy; print(scipy.__file__)"``.
 
-   - The interactive examples are not available in CI builds. To see the
-     interactive examples locally in a JupyterLab environment, you can use
-
-     .. code-block:: bash
-
-        python -m http.server --directory doc/build/html
-
-     The documentation pages should then be available at `http://localhost:8000/`.
+   - The SciPy documentation contains interactive examples rendered with the
+     help of the `jupyterlite-sphinx` extension. This functionality may not be
+     enabled for local builds of the documentation, or in docs CI previews. For
+     more details, see :ref:`interactive-docs`.
```

Comment:
As long as both `jupyterlite-sphinx` and a kernel via `jupyterlite-pyodide-kernel` are installed, the interactive docs functionality will always be enabled in local builds and docs CI previews (unless there's something I'm missing?).

Reply:
You are right, I think this is just outdated. Removed